File: Logitude/Simplog.Server.Infrastructure/Helpers/SystemEnvironmentService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Web;

namespace Simplog.Server.Infrastructure.Helpers
{
    public class SystemEnvironmentService
    {

        public static string GetLoggedDomain()
        {
            HttpContext context = HttpContext.Current;
            string Url = context.Request.Url.ToString().Split('/')[2];//("http://", "");
            Url = Url.Split(':')[0]; 
            return Url;
        }

        public static bool IsLogBox()
        {
            string url = GetLoggedDomain();
            if (!string.IsNullOrEmpty(LogitudeSettings.DeploymentStage) && (LogitudeSettings.DeploymentStage.ToLower() == "logboxpre" || LogitudeSettings.DeploymentStage.ToLower() == "logboxwe1" || LogitudeSettings.DeploymentStage.ToLower() == "test2"))
            {
                if (url.Contains("system.logbox.co.il") || url.Contains("test.logitudeworld.com") || url.Contains("pre.logbox.co.il"))
                {
                    return true;
                }
            }
            return false;
        }
         
    }
}

File: Logitude/AngularModules/AngularModules/Infrastructure/Utilities/SystemEnvironmentService.ts
import { ObjectsLocator } from '../Locators/ObjectsLocator';
import { AppTool } from '../Tools';
import { SessionLocator } from './SessionLocator';

export class SystemEnvironmentService {

    public static GetLogitudeURL() {
        return AppTool.GetLogitudeURL();
    }

    public static IsLogBox(): boolean {
        let isPrivateLabel = SessionLocator.PrivateLableSettings ? true : false;
        let url: string = this.GetLogitudeURL().toLowerCase();

        if (ObjectsLocator.GlobalSetting.DeploymentStage == "logboxwe1" || ObjectsLocator.GlobalSetting.DeploymentStage == "Test2" || ObjectsLocator.GlobalSetting.DeploymentStage == "Dev") {
            if ((url.indexOf("logbox") > -1 || url.indexOf("test.logitudeworld.com") > -1 || url.indexOf("localhost:9996") > -1) && !isPrivateLabel) {
                return true;
            }
        }
        return false;
    }

}



